src/core/index.ts

```diff
@@ -93,57 +101,117 @@ export interface PromptUiActions {
     };
 }
 
+export interface UnderlayConfig {
+    colors: {
+        bg: string;
+    };
+}
+
+export enum PromptPosition {
+    TOP_LEFT = 'top-left',
+    TOP_CENTER = 'top-center',
+    TOP_RIGHT = 'top-right',
+    CENTER_LEFT = 'center-left',
+    CENTER = 'center',
+    CENTER_RIGHT = 'center-right',
+    BOTTOM_LEFT = 'bottom-left',
+    BOTTOM_CENTER = 'bottom-center',
+    BOTTOM_RIGHT = 'bottom-right',
+    TOP = 'top',
+    BOTTOM = 'bottom'
+}
+
 interface BasePromptConfig {
     uuid: string;
-    type: string;
+    type: PromptTypeName;
     trigger: PromptTrigger;
-    position: string;
+    position: PromptPosition;
+    labels: any;
+    colors: any;
     overlay?: PromptOverlayConfig;
-    actions?: PromptAction[];
+    actions: any[];
```

Comment:
Why did we make this non-optional (would this be a breaking iface change for existing consumers?), and why now `any` type?

Reply:
Must have done this while refactoring/debugging and meant to flip it back! fixed!